packages/components/tests/dummy/app/templates/application.hbs

```diff
@@ -20,14 +20,43 @@
     {{outlet}}
   </main>
   <footer class="dummy-footer">
-    <p class="dummy-paragraph">
-      <a
-        class="dummy-link"
-        href="https://github.com/hashicorp/design-system/tree/main/packages/components"
-        target="_blank"
-        rel="noopener noreferrer"
-      >GitHub repo
-      </a>
-    </p>
+    <div class="dummy-row-wrapper">
+      <div class="dummy-row">
+        <p class="dummy-paragraph-title">Our Accessibility Statement</p>
+        <p class="dummy-paragraph">
+          Each of our components intends to conform to WCAG 2.1 AA standards (at a minimum). If you find an issue with
+          any component, you can let us know in any of the following ways:
+        </p>
+        <ol class="dummy-list">
+          <li>
+            Post a message in the #team-design-systems channel in our internal Slack
+          </li>
+          <li>
+            Email
```

Comment:
non-blocking suggestion: Would probably put filing an issue above email in terms of preference on our end / consumer end?

Reply:
In terms of "ways to contact" we should consider the preferences of the user who has accessibility needs and wants to report an issue. Email is preferred over GitHub for those use cases, which is why I've put it before GitHub. 👍